test/cmd/sdn.sh

```diff
@@ -15,47 +15,47 @@ trap os::test::junit::reconcile_output EXIT
 os::test::junit::declare_suite_start "cmd/sdn"
 
 os::test::junit::declare_suite_start "cmd/sdn/clusternetworks"
-os::cmd::expect_success 'oc get clusternetworks'
-# Sanity check that the environment is as expected, or the NetNamespace tests will fail
-os::cmd::expect_success_and_text 'oc get clusternetwork default -o jsonpath="{.pluginName}"' 'redhat/openshift-ovs-multitenant'
-os::test::junit::declare_suite_end
-
-os::test::junit::declare_suite_start "cmd/sdn/netnamespaces"
-orig_project="$(oc project -q)"
-
-os::cmd::expect_success 'oc get netnamespaces'
-os::cmd::expect_success_and_text 'oc get netnamespace default -o jsonpath="{.netid}"' '^0$'
-
-os::cmd::expect_success 'oc new-project sdn-test-1'
-os::cmd::expect_success 'oc new-project sdn-test-2'
-os::cmd::expect_success 'oc new-project sdn-test-3'
-os::cmd::try_until_success 'oc get netnamespace sdn-test-1'
-os::cmd::expect_success_and_not_text 'oc get netnamespace sdn-test-1 -o jsonpath="{.netid}"' '^0$'
-orig_vnid1="$(oc get netnamespace sdn-test-1 -o jsonpath='{.netid}')"
-os::cmd::try_until_success 'oc get netnamespace sdn-test-2'
-os::cmd::try_until_success 'oc get netnamespace sdn-test-3'
-
-os::cmd::expect_success 'oc adm pod-network join-projects --to=sdn-test-1 sdn-test-2'
-os::cmd::expect_success_and_text 'oc get netnamespace sdn-test-2 -o jsonpath="{.netid}"' "^${orig_vnid1}\$"
-
-os::cmd::expect_success 'oc adm pod-network make-projects-global sdn-test-1'
-os::cmd::expect_success_and_text 'oc get netnamespace sdn-test-1 -o jsonpath="{.netid}"' '^0$'
-
-os::cmd::expect_success 'oc adm pod-network isolate-projects sdn-test-1'
-os::cmd::expect_success_and_not_text 'oc get netnamespace sdn-test-1 -o jsonpath="{.netid}"' '^0$'
-
-os::cmd::expect_success "oc project '${orig_project}'"
-os::test::junit::declare_suite_end
-
-os::test::junit::declare_suite_start "cmd/sdn/hostsubnets"
-# test-cmd environment has no nodes, hence no hostsubnets
-os::cmd::expect_success_and_text 'oc get hostsubnets' 'No resources found.'
-os::test::junit::declare_suite_end
-
-os::test::junit::declare_suite_start "cmd/sdn/egressnetworkpolicies"
-os::cmd::expect_success 'oc get egressnetworkpolicies'
-os::cmd::expect_success 'oc create -f test/integration/testdata/test-egress-network-policy.json'
-os::cmd::expect_success 'oc get egressnetworkpolicy default'
+#os::cmd::expect_success 'oc get clusternetworks'
```

Comment:
Issue to get them back or remove if not needed, I don't see any clear information about it anywhere.
/cc @squeed

Reply:
Ah I deleted this whole file. Linking Dan's comment: https://github.com/openshift/origin/pull/21206#discussion_r225298568

Reply:
@deads2k Is this test okay to remove / do we need to restore it back at some point?

Reply:
delete the entire file and open a high severity bugzilla for networking team

Reply:
@knobunc @squeed with the network APIs gone, we can no longer run these tests in test-cmd.sh.  @juanvallejo will open a bugzilla and reference the commit he has to delete them

Reply:
Opened high severity bugzilla here: https://bugzilla.redhat.com/show_bug.cgi?id=1639835